pkg/standalone/stop_windows.go

```diff
@@ -21,7 +21,12 @@ func Stop(appID string) error {
 
 	for _, a := range apps {
 		if a.AppID == appID {
-			eventName, _ := syscall.UTF16FromString(fmt.Sprintf("dapr_cli_%v", a.PID))
+			pid := fmt.Sprintf("dapr_cli_%v", a.CliPID)
+			if pid == "0" {
+				pid = fmt.Sprintf("dapr_cli_%v", a.DaprdPID)
```

Comment:
So I'm pretty sure this won't work unless we modify daprd to listen for this event. We can do that, of course, but daprd doesn't support this on windows today.

Reply:
@sk593 Can you create an issue in dapr/dapr to add this event listen to daprd windows?

Reply:
Sure, just made one [here](https://github.com/dapr/dapr/issues/3524)